substrate/primitives/keyring/src/ed25519.rs

```diff
@@ -134,6 +135,30 @@ impl From<Keyring> for sp_runtime::MultiSigner {
 	}
 }
 
+impl FromStr for Keyring {
+	type Err = ParseKeyringError;
+
+	fn from_str(s: &str) -> Result<Self, <Self as FromStr>::Err> {
+		match s {
+			"Alice" | "alice" => Ok(Keyring::Alice),
+			"Bob" | "bob" => Ok(Keyring::Bob),
+			"Charlie" | "charlie" => Ok(Keyring::Charlie),
+			"Dave" | "dave" => Ok(Keyring::Dave),
+			"Eve" | "eve" => Ok(Keyring::Eve),
+			"Ferdie" | "ferdie" => Ok(Keyring::Ferdie),
+			"Alice//stash" => Ok(Keyring::AliceStash),
```

Comment:
```suggestion
			"Alice//stash" | "alice//stash => Ok(Keyring::AliceStash),
```
Same below and in `sr25519` module